model.go

```diff
@@ -89,6 +89,12 @@ type Model interface {
 	AddRemoteApplication(RemoteApplicationArgs) RemoteApplication
 
 	Validate() error
+
+	SetSLA(level string, credentials []byte)
```

Comment:
All of the other set methods also return the new thing set. So `SetSLA` should return the `SLA`, and `SetMeterStatus` should return the `MeterStatus`